From c78a857c027806b67ddd1c8a32cf2ca3d5c047f8 Mon Sep 17 00:00:00 2001
From: Moritz Warning <moritzwarning@web.de>
Date: Wed, 23 Sep 2015 20:38:29 +0200
Subject: [PATCH 02/25] give modloader more time to load modules

otherwise wifi detect will not detect all wifi cards (e.g for TP-Link Archer)
and /etc/config/wireless will not be fully initialized
---
 package/base-files/files/etc/init.d/boot | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package/base-files/files/etc/init.d/boot b/package/base-files/files/etc/init.d/boot
index a53be74..771cf38 100755
--- a/package/base-files/files/etc/init.d/boot
+++ b/package/base-files/files/etc/init.d/boot
@@ -39,7 +39,7 @@ boot() {
 	/sbin/kmodloader
 
 	# allow wifi modules time to settle
-	sleep 1
+	sleep 3
 
 	/sbin/wifi detect > /tmp/wireless.tmp
 	[ -s /tmp/wireless.tmp ] && {
-- 
2.1.4

